cpp/lib/src/ZoteroHarvesterConversion.cc

```diff
@@ -541,12 +553,7 @@ void DetectLanguage(MetadataRecord * const metadata_record, const Config::Journa
             record_text = metadata_record->title_ + " " + metadata_record->abstract_note_;
         else
             LOG_ERROR("unknown text field '" + journal_params.language_params_.source_text_fields_ + "' for language detection");
-
-        std::vector<NGram::DetectedLanguage> detected_languages;
-        NGram::ClassifyLanguage(record_text, &detected_languages, journal_params.language_params_.expected_languages_,
-                                /*alternative_cutoff_factor = */ 0);
-        const auto top_language(detected_languages.front());
-        detected_language = top_language.language_;
+        detected_language = TikaDetectLanguage(record_text);
```

Comment:
https://www.tutorialspoint.com/tika/tika_language_detection.htm

It seems like Tika only supports 18 languages, and some of the languages we already use in zotero_harvester.conf are not supported (afr, ara, may, ron, srp, tur). 

Maybe we need to use NGram as fallback if Tika cannot detect any language?